packages/main/src/RatingIndicator.hbs

```diff
@@ -46,35 +21,23 @@
 		>
 		{{#each _stars}}
 			{{#if this.selected}}
-				<li @click="{{../_onclick}}" data-ui5-value="{{this.index}}" class="ui5-rating-indicator-item ui5-rating-indicator-item-sel">
-					<ui5-icon name="favorite"></ui5-icon>
+				<li data-ui5-value="{{this.index}}" class="ui5-rating-indicator-item ui5-rating-indicator-item-sel">
+					<ui5-icon data-ui5-value="{{this.index}}" name="favorite"></ui5-icon>
 				</li>
 			{{else if this.halfStar}}
-				<li @click="{{../_onclick}}" data-ui5-value="{{this.index}}" class="ui5-rating-indicator-item ui5-rating-indicator-item-half">
-					{{#if this.isFiori}}
-						{{> fioriSvg}}
-					{{else}}
-						{{> horizonSvg}}
-					{{/if}}
+				<li class="ui5-rating-indicator-item ui5-rating-indicator-item-half">
+					<ui5-icon data-ui5-value="{{this.index}}" name="unfavorite"></ui5-icon>
 					<div class="ui5-rating-indicator-half-icon-wrapper">
-						<ui5-icon name="favorite" class="ui5-rating-indicator-half-icon"></ui5-icon>
+						<ui5-icon data-ui5-value="{{this.index}}" name="favorite" class="ui5-rating-indicator-half-icon"></ui5-icon>
 					</div>
 				</li>
 			{{else if ../readonly}}
-				<li @click="{{../_onclick}}" data-ui5-value="{{this.index}}" class="ui5-rating-indicator-item ui5-rating-indicator-readonly-item-unsel">
-					{{#if this.isFiori}}
-						{{> fioriSvg}}
-					{{else}}
-						{{> horizonSvg}}
-					{{/if}}
+				<li class="ui5-rating-indicator-item ui5-rating-indicator-readonly-item-unsel">
+					<ui5-icon data-ui5-value="{{this.index}}" name="unfavorite"></ui5-icon>
```

Comment:
And, don't forget to import the icons

import "@ui5/webcomponents-icons/dist/favorite.js";
import "@ui5/webcomponents-icons/dist/unfavorite.js";

Reply:
Done.